examples/react-native/package.json

```diff
@@ -19,9 +19,12 @@
     "@react-native-async-storage/async-storage": "^1.17.5",
     "@react-native-community/netinfo": "^8.3.0",
     "@xstate/react": "3.0.0",
+    "amazon-cognito-identity-js": "^5.2.11",
```

Comment:
For my own understanding, for rn are we not taking a dependency on the umbrella `aws-amplify` package?